My Simple App/Business Logic/Event.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+
+/// One to many communication of a value's updates.
+///
+/// # Example
+/// ```
+/// class KeyboardListner {
+///   let keyboardHeight: Event<CGFloat>
+///
+///   ...
+/// }
+///
+/// class MyViewController {
+///   let keyboard = KeyboardListner()
+///
+///   init() {
+///     self.keyboard.keyboardHeight.add(subscriber: self) { [weak self] newHeight in
+///       self?.updateUIToNewKeyboardHeight(newHeight)
+///     }
+///   }
+/// }
+/// ```
+class Event<Value> {
```

Comment:
woodward4422 2 days ago @woodward4422
Seems like you are doing a reactive paradigm similar to Combine, why not use combine then?

LinnierGames 34 minutes ago Author Owner
Since Combine hasn't been used in our codebase and it's not something SUPER familiar with most iOS devs, I decided not to use Combine but I do agree this would be a great application for Combine